python-lib/translate.py

```diff
@@ -158,7 +158,7 @@
     "bs": "bg",
     "ca": "es",
     "ceb": "en",  # Not same family, but Latin script
-    "cz": "sk",
```

Comment:
We should keep the incorrect one for backwards compatibility though

Reply:
good shout